pkg/client/edv/client_test.go

```diff
@@ -60,6 +64,23 @@ func (f failingReadCloser) Close() error {
 	return fmt.Errorf("failingReadCloser always fails")
 }
 
+type TestLoggerProvider struct {
```

Comment:
do we need it

Reply:
@sudeshrshetty do we have it in aries

Reply:
I don't think we have it in aries,
@DRK3 why this one is being added?

Reply:
@fqutishat Some of the unit tests I had before I could only effectively test by reading into the log - this TestLoggerProvider allows me to read into the log now with the new package. Unless there's another way? @sudeshrshetty